lib/sensimity.js

```diff
@@ -22,12 +22,20 @@ if (_.isUndefined(Alloy.Globals.sensimityEvent)) {
  * Initialize the scanner and start scanning on added network identifier
  */
 function start(options) {
-    if (_.isUndefined(Alloy.Globals.sensimityScanner)) {
-        Alloy.Globals.sensimityScanner = createScanner(options);
-        initScannerAndStartScanning(options);
-    } else {
-        Ti.API.warn('Scanner already defined, please destruct first before start scanning');
-    }
+    // Only start Sensimity when bluetooth is enabled
+    isBLEEnabled(function (value) {
```

Comment:
One should be able to add in a callback function on the start when things go wrong. Now you have the ability to add your own BLE checks, but this will result in double checks before really starting sensimity.